src/web/gwc/src/main/java/org/geoserver/gwc/web/layer/CachedLayersPage.java

```diff
@@ -323,9 +329,77 @@ protected Component headerPanel() {
         removal.setOutputMarkupId(true);
         removal.setEnabled(false);
 
+        // the clear All GWC cache link
+        header.add(trucateAllLink("clearGwcLink"));
+
         return header;
     }
 
+    private AjaxLink<String> trucateAllLink(String id) {
+        AjaxLink<String> trunkAlllink =
+                new AjaxLink<String>(id) {
+
+                    @Override
+                    public void onClick(AjaxRequestTarget target) {
+                        dialog.setTitle(
+                                new ParamResourceModel(
+                                        "confirmGwcTruncateTitle", CachedLayersPage.this));
+                        dialog.setDefaultModel(getDefaultModel());
+
+                        dialog.showOkCancel(
+                                target,
+                                new GeoServerDialog.DialogDelegate() {
+                                    private static final long serialVersionUID = 1L;
+                                    private TruncateAllRequest truncateAllRequest;
+
+                                    @Override
+                                    protected Component getContents(final String id) {
+                                        Label confirmLabel =
+                                                new Label(
```

Comment:
This should be internationalized (like all UI messages)

Reply:
will do